docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+The Strapi docs are power by [VuePress](https://vuepress.vuejs.org/). The docs live in a folder called **docs**. `Path: ./strapi/docs`. The _VuePress_ installation lives in this folder. It has a separate installation and separate dependencies from the rest of the Strapi monorepo.
+
+Only version **3.0.0-beta.x** is currently maintained. So, therefore, only the documentation for this version is maintained. The documentation is located at `./strapi/docs/3.0.0-beta.x/`. Here you find all the content and assets for the documentation.
+
+The _folder names_ in the file directory correspond to the different sections in the left-hand menu. For example, the folder called, "Getting Started," corresponds to the section called, "Getting Started." However, the `h1` tag at the top of the files control the left-menu titles.
+
+For example, if you add a page or folder, you do it in the config file of the Beta docs. `Path: ./strapi/docs/.versions/3.0.0-beta.x.js/`
+
+For example, if you wanted to add a page under `Getting Started` and the file is called `example-tutorial.md`, you would save the file in the folder called `/getting-started/`. Next, you would modify the config file like the example below:
+
+`Path: ./strapi/docs/.versions/3.0.0-beta.x.js/`
+
+```js
+module.exports = [
+  {
+    collapsable: false,
+    title: '🚀 Getting started',
+    children: [
+      '/3.0.0-beta.x/getting-started/introduction',
+      '/3.0.0-beta.x/getting-started/install-requirements',
+      '/3.0.0-beta.x/getting-started/quick-start',
+      '/3.0.0-beta.x/getting-started/quick-start-tutorial',
+      '/3.0.0-beta.x/getting-started/example-tutorial',  // <- Added here
+    ],
+  },
+
+```
+
+For example, if you now wanted the **left-hand** menu to have a link to **Example Tutorial**, you would need to add an `h1` tag with `Example Tutorial`, like the below example:
+
+```md
+# Example Tutorial
+
+This is an Example Tutorial....
+```
+
+You should understand that [VuePress]() automatically turns `h2` and `h3` tags into correctly formed **submenu items** and **nested submenu items**. These display in the left-hand navigation underneath the correct article heading as determined by the `h1` tag on the page.
+
+Now, if you need to add images to the file, there is a folder called `assets`, the `Path: ./3.0.0-beta.x/assets`. When you add images, these images need are added to a new folder in the `assets` folder. The name of the folder, should match the name of the file and be placed in a matching directory to the directory of the file.
+
+For example, the `example-tutorial.md` file is located in the `/getting-started/` directory.
+
+```js
+module.exports = [
+  {
+    collapsable: false,
+    title: '🚀 Getting started',
+    children: [
+      '/3.0.0-beta.x/getting-started/example-tutorial',  // <- Added here
+    ],
+  },
+
+```
+
+Therefore, images are placed in `./assets/getting-started/example-tuturial/image.png`, like this:
+
+```
+DOCS
```

Comment:
I think that the schema below is really to understand maybe you can simply use it instead of detailing how it works.

What do you think?

Reply:
I totally simplified the earlier explanation before the schema.  It should be clear now.